README.md

```diff
@@ -89,6 +89,11 @@ bucket identifier (number)
     - Calculate bucket id for a specified space and sharding key.
     Method uses sharding function specified in DDL schema.
 
+    Method is not transactional in the sense that it catches up
+    _ddl_sharding_func changes immediatelly: it may see changes that're
```

Comment:
Nit: Usually, we mark words related to source code with apostrophes for better visibility.
This one `_ddl_sharding_func` is looking better than "_ddl_sharding_func". However, it may look good for your taste, so I will not insist.

Reply:
fixed

Reply:
(Looks resolved, so I'm going to resolve this discussion.)